phpunit.xml.dist

```diff
@@ -18,15 +19,18 @@
 
     <php>
         <var name="driver_config_factory" value="Mink\WebdriverClassDriver\Tests\WebdriverClassicConfig::getInstance"/>
-        <!--server name="WEB_FIXTURES_HOST" value="http://test.mink.dev" /-->
+
+        <server name="WEB_FIXTURES_HOST" value="http://host.docker.internal:8002"/>
         <!-- MacOS -->
         <!--<server name="WEB_FIXTURES_HOST" value="http://docker.for.mac.localhost:8002"/>-->
-        <!--<server name="WEB_FIXTURES_BROWSER" value="firefox"/>-->
+
         <!-- where driver will connect to -->
         <server name="DRIVER_URL" value="http://localhost:4444/wd/hub"/>
+
         <!-- where DocumentRoot of 'Test Machine' is mounted to on 'Driver Machine' (only if these are 2 different machines) -->
-        <!--server name="DRIVER_MACHINE_BASE_PATH" value="" /-->
-        <!--server name="TEST_MACHINE_BASE_PATH" value="" /-->
+        <server name="DRIVER_MACHINE_BASE_PATH" value="/fixtures/"/>
+
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="disabled" />
```

Comment:
@stoff I wrote my thoughts [on this already](https://github.com/minkphp/MinkSelenium2Driver/issues/366#issuecomment-1585666969), but let's maybe let's start a proper discussion here?

Maybe I'm missing something, but isn't *display errors* + *enabling all errors* + *disallowing output from tests* good enough (and maybe also the expected flow)?

Reply:
I don't think what you describe will be enough.
Also, symfony/phpunit-bridge enables the DebugClassLoader when available, which reports additional deprecations based on class definitions.
I would prefer if this repo was using the listener of the bridge, as done in other drivers (note that we are not using the simple-phpunit that manages a separate phpunit installation. We are still installing phpunit as a composer dependency).

Reply:
ok then , I'll add it back 👍